recipe/meta.yaml

```diff
@@ -18,13 +18,17 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - libgcc-ng {{ libgcc }}
```

Comment:
These pins are added only for x86 in conda_build_config.yaml. So, we need #[x86_64] here too.

Reply:
I updated https://github.com/open-ce/open-ce/pull/314 to include the pin for ppc64le as well.

Reply:
Also we don't have a branch for 1.1.x, only tags. So we can push this to master, then branch it when needed. We may want to control these pins on master going forward as well anyway.